docs/community.md

```diff
@@ -0,0 +1,68 @@
+# Code of Conduct
+This community follows the [{Code} of
+Conduct](https://github.com/codedellemc/community/blob/master/code-of-conduct.md).
+
+---
+
+# Roles and Responsibilities
+Going top down, each role inherits and extends documented responsibilities.
+
+## {code} Community Member
+Community Members are interested in learning about, using and contributing to
+the {code} open source projects. A Community Member adheres to Code of Conduct.
+
+**Learn More**
+
+* [{Code} Community](https://thecodeteam.com/community/)
+* [{Code} Projects](https://thecodeteam.com/all-projects/)
+
+## User
+Users read our documentation and use our software; they provide feedback to the
+community and share their experience (videos, blogs etc.) on their usage of the
+software.  A User enters a bug report or enhancement request into the project's
+tracking system, helps new users and recommends the project to others.
+
+**Learn More** 
+
+* [Product Documentation TBD](tbd.md)
+* [Project issue tracker and filing process](contributing.md#issues-and-guidance)
+* [Communication channels](contributing.md#support-channel)
+
+## Developer
+Developers contribute changes to the project (such as bug fixes, enhancements,
+documentation updates, tests, tools and any other improvements to the project).
+A Developer helps in debugging and triaging issues filed and responds to any
+questions posed by the community against the component/project.
+
+**Learn More**
+
+* [Developer guide (includes how to contribute and develop code) TBD link to contributing](tbd.md)
+
+## Reviewer
+A Reviewer has been contributing to a component significantly and holds a
+deeper understanding of the component and tools.  Reviewers assess
+contributions and provide feedback and guidance to developers; they have access
+to the project backlog and can vote on backlog item priority.
+
+**Learn More**
+
+* [Becoming a reviewer TBD](tbd.md)
+
+## Maintainer
+Maintainers are responsible for the strategic interests of the component and
+the project. A Maintainer reviews code contributions and accepts pull requests,
+continuously grooms the backlog and balances the project's long-term goals
+while managing timely releases.  Maintainers take and provide feedback to
+improve existing processes and create new ones.
+
+**Learn More**
+
```

Comment:
Given the exercise we went through yesterday, we should add a hook/link to a page that describes the required project and tools setup permission process. I would envision a maintainer of a project would need to do this.